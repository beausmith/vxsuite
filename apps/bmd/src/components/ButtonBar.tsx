import styled from 'styled-components'

const ButtonBar = styled.div`
  display: flex;
  padding: 1rem 2rem;
  background: grey;
  border-bottom: 1px solid darkgrey;
  justify-content: space-between;
  & > :first-child {
    order: 2;
  }
  @media print {
    display: none;
  }
`

export default ButtonBar
