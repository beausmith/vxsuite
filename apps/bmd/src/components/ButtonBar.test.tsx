import { axe } from 'jest-axe'
import React from 'react'
import { render } from 'react-testing-library'

import ButtonBar from './ButtonBar'

it(`renders ButtonBar`, async () => {
  const { container } = render(<ButtonBar>foo</ButtonBar>)
  expect(container.firstChild).toMatchSnapshot()
  expect(await axe(container.innerHTML)).toHaveNoViolations()
})
