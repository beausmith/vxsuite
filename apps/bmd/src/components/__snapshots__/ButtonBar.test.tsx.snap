// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ButtonBar 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  padding: 1rem 2rem;
  background: grey;
  border-bottom: 1px solid darkgrey;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

.c0 > :first-child {
  -webkit-order: 2;
  -ms-flex-order: 2;
  order: 2;
}

@media print {
  .c0 {
    display: none;
  }
}

<nav
  class="c0"
>
  foo
</nav>
`;
